internal/proposals/logs/improvement-of-log-messages-usability.md

```diff
@@ -34,6 +34,8 @@ To unify the logs, which will make the debugging process and logs parsing much e
     - error and fatal message: past tense started with, for example, `Failed to...`, after that the error wrapped with some meaningful context. For example: `Failed to provision runtime: while fetching release: while validating release: release does not contain installer yaml`.
     - info message: present continuous tense for the things that are about to be done, for example, `Starting processing...`, or past tense for the things that are finished, such as `Finished successfully!`.
     - notice and warning message: a short explanation of what happened and what this can cause. For example: `Tiller configuration not found in the release artifacts. Proceeding to the Helm 3 installation...` or `Connection is not yet established. Retrying in 5 minutes...`.
```

Comment:
```suggestion
    - warning message: a short explanation of what happened and what this can cause. For example: `Tiller configuration not found in the release artifacts. Proceeding to the Helm 3 installation...` or `Connection is not yet established. Retrying in 5 minutes...`.
```